README.md

```diff
@@ -7,53 +7,78 @@ https://circleci.com/orbs/registry/orb/mgmorbs/microservices
 
 ### Simple Usage with Workflows
 
-This is an example of a drop-in CircleCI configuration that can be placed at `.circleci/config.yml` for any microservice for out-of-the-box builds.
+This is an example of a drop-in CircleCI configuration that can be placed at `.circleci/config.yml` for any microservice for (almost) out-of-the-box builds.
+
+This Orb depends on each service to implement `build tag export-image push-image-to-ecr ci-test-unit ci-test-integration ci-lint` make targets.
 
 ```yml
 version: 2.1
 
 orbs:
-  microservices: mgmorbs/microservices@0.3
+  ms: mgmorbs/microservices@1
 
 workflows:
   version: 2
   build_and_test:
     jobs:
-      # Step 1: Fetches env config and prints diagnostic info
-      - microservices/initial-setup:
+      - ms/setup-env:
+          context: microservices
+          filters:
+              tags:
+                only: /^v.*/
```

Comment:
@misiek-sf Can you elaborate on why we're only building on tags and not on every commit of filtered branches?